src/host/mapreduce.jl

```diff
@@ -12,8 +12,8 @@ Base.mapreducedim!(f, op, R::AnyGPUArray, A::Broadcast.Broadcasted) = mapreduced
 
 neutral_element(op, T) =
     error("""GPUArrays.jl needs to know the neutral element for your operator `$op`.
-             Please pass it as an explicit argument to (if possible), or register it
-             globally your operator by defining `GPUArrays.neutral_element(::typeof($op), T)`.""")
+             Please pass it as an explicit argument to `GPUArrays.mapreducedim!` (if possible),
+             or register it globally your operator by defining `GPUArrays.neutral_element(::typeof($op), T)`.""")
```

Comment:
```suggestion
             or register it globally for your operator by defining `GPUArrays.neutral_element(::typeof($op), T)`.""")
```

i'm unsure if this is the correct wording, but something still needs reworded here i think.

Reply:
Other than these suggested rewordings, it looks good to me - thanks!